nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/service/StandardControllerServiceProvider.java

```diff
@@ -70,7 +67,7 @@
 
     private final ProcessScheduler processScheduler;
     private static final Set<Method> validDisabledMethods;
-    private final BulletinRepository bulletinRepo;
+    // private final BulletinRepository bulletinRepo;
```

Comment:
I removed the two commented lines (the declaration and the assignment in the constructor). However, I did not change the constructor contract, as it is used 17 times throughout the application. @mcgilman @markap14 is the bulletin repository work going to have additional impact, or is it safe to clean up this constructor universally?